CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 5.4.1
+- Correctly perform healthchecks against ES 5.x nodes
```

Comment:
Hah, I recently changed the healthcheck in a previous PRe and got some wires crossed. Good catch :)